addons/mail_bot/static/src/owl/components/notification_request/notification_request.js

```diff
@@ -0,0 +1,72 @@
+odoo.define('mail_bot.component.NotificationRequest', function (require) {
+'use strict';
+
+const PartnerImStatusIcon = require('mail.component.PartnerImStatusIcon');
+const useStore = require('mail.hooks.useStore');
+
+var { _t } = require('web.core');
```

Comment:
prefer accessing `_t` from `env`

Reply:
We should do a pass to replace all the others or we'll keep copying them wrongly.